### DISCUSSION TITLE
Always on location tracking

### DISCUSSION THREAD

**Discussion by carmolim** [TICESoftware/TICE#8] [Ideas]
I was using Life360 with my family members, but after the news about the selling of location data I started to look for alternatives, I really liked what you done with Tice in terms of privacy, but I would like to know if you have plans to add background location tracking, and maybe some geofences with alerts. Thanks!

**Comment by FabioTacke** [comment]
Hi @carmolim! Thank you for your positive feedback!
Permanent location sharing and geofence based alerts have been on our feature list right from the beginning. But we wanted to focus on the use case of ad-hoc meetings first. As the TICE app is rather a side project at the moment for us we only have capacities for maintenance and bug fixing right now. However this may change in the future and we hope to be able to invest more time into TICE again sometime.
That begin said, as soon as the backend will be open-sourced contributing to the project will be much easier and anyone could help us integrating additional features much quicker.
